setup.py

```diff
@@ -53,6 +58,7 @@
         'console_scripts': [
             'make_composite = asf_tools.composite:main',
             'water_map = asf_tools.water_map:main',
+            'calculate_hand = asf_tools.hand:main',
```

Comment:
I struggled remembering the verb was `calculate`, suggesting it's not a great name, but I'm not sure what's better.  `make` mimics what we did for composites.